pkg/consoleplugin/deploy/deployment_test.go

```diff
@@ -159,22 +159,22 @@ func unmarshalObj(t *testing.T, content string, target runtime.Object) {
 
 // contains an empty spec because we do not verify the actual spec value
 func service(namespace string) string {
-	return fmt.Sprintf(`{"apiVersion":"v1","kind":"Service","metadata":{"labels":{"provider":"codeready-toolchain","run":"member-operator-console-plugin"},"name":"member-operator-console-plugin","namespace":"%s"},"spec":{}}`, namespace)
+	return fmt.Sprintf(`{"apiVersion":"v1","kind":"Service","metadata":{"labels":{"toolchain.dev.openshift.com/provider":"codeready-toolchain","run":"member-operator-console-plugin"},"name":"member-operator-console-plugin","namespace":"%s"},"spec":{}}`, namespace)
 }
```

Comment:
do these tests make any sense when we just need to copy-paste the same content that we put into the manifest file? It reminds me the unit tests we had for tiers and we dropped because it was hard to maintain and didn't bring any value.

Reply:
Yep. I had the same thoughts.